airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -26,7 +26,8 @@
 import attr
 from google.api_core.exceptions import Conflict
 from google.api_core.retry import Retry
-from google.cloud.bigquery import DEFAULT_RETRY, CopyJob, ExtractJob, LoadJob, QueryJob
+from google.cloud.bigquery import DEFAULT_RETRY, CopyJob, ExtractJob, LoadJob, QueryJob, UnknownJob
```

Comment:
If UnknownJob is used only for annotation, we can have it under "if TYPE_CHECKING:"